CLA.md

```
@@ -0,0 +1,171 @@
+project carried when those Contributions were submitted, and nothing in this
+Agreement is intended to change that. This Agreement does not modify the license
+of the Work itself, which is stated in [LICENSE](LICENSE).
+
+---
+
+## Signature record
+
+Signatures are recorded by the CLA assistant bot on the pull request. An out of
+band signature must supply:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)